source/common/http/conn_manager_impl.cc

```diff
@@ -857,10 +859,20 @@ void ConnectionManagerImpl::ActiveStream::decodeData(ActiveStreamDecoderFilter*
   if (!filter) {
     entry = decoder_filters_.begin();
   } else {
-    entry = std::next(filter->entry());
+    if ((*(filter->entry()))->stoppedAll()) {
+      // If filter's iteration_state_ has been stopped, its decodeData() has not be called. Call it
+      // now.
```

Comment:
This only happens in the continue case, right? Can you clarify that in the comment?

Reply:
That's right! Added comments to clarify.